documentation/NUnit2048.md

```diff
@@ -0,0 +1,101 @@
+# NUnit2048
```

Comment:
Somehow I've ended up pushing with windows line endings (causing the appveyor build to fail), no idea how.

Reply:
Check your git settings for `git core.autocrlf`
We should probably fix this in a .got attributes so it doesn't depend on a user's settings

Reply:
Or not?  Everything seems happy now.  Not sure what happened.